### ISSUE TITLE
PyPi download missing numbers.format_compact_currency

### ISSUE THREAD

**Issue by typonaut** [python-babel/babel#964]
The PyPi package for Babel appears to be missing ```babel.numbers.format_compact_currency```.

I have checked the archive available for download directly, and the ```babel.numbers.py``` file from PyPi is 1,165 lines, compared to the 1,323 in the archive available here.

Both downloads report that they are version ```2.11.0```

**Comment by jun66j5** [comment]
Incorrect.

`babel/numbers.py` file in v2.11.0 has 1,164 lines. Also, wheel and tarball files for v2.11.0 on pypi have the same lines.

```
$ curl -sL https://github.com/python-babel/babel/raw/v2.11.0/babel/numbers.py | wc -l
1164
$ pip download -d /tmp 'Babel==2.11.0'
$ pip download --no-binary=:all: -d /tmp 'Babel==2.11.0'
$ tar xaf /tmp/Babel-2.11.0.tar.gz -O Babel-2.11.0/babel/numbers.py | wc -l
1164
$ unzip -x -p /tmp/Babel-2.11.0-py3-none-any.whl babel/numbers.py | wc -l
1164
```

The `format_compact_currency` method is added in 5fcc2535f96bfce9c1a1ecf9d19a976b9bf6ab6b and not included in 2.11.0 (not released yet).

**Comment by akx** [comment]
Yep, 2.11 hasn't been released yet.

Thanks for verifying, @jun66j5 🙇
